src/modules/reserve-overview/ReserveFactorOverview.tsx

```diff
@@ -0,0 +1,83 @@
+import { ExternalLinkIcon } from '@heroicons/react/outline';
+import { Trans } from '@lingui/macro';
+import { Box, SvgIcon, Typography } from '@mui/material';
+import { ReserveFactorTooltip } from 'src/components/infoTooltips/ReserveFactorTooltip';
+import { FormattedNumber } from 'src/components/primitives/FormattedNumber';
+import { Link } from 'src/components/primitives/Link';
+import { ReserveOverviewBox } from 'src/components/ReserveOverviewBox';
+import { ExplorerLinkBuilderProps } from 'src/ui-config/networksConfig';
+
+interface ReserveFactorOverviewProps {
+  collectorContract: string;
+  explorerLinkBuilder: (props: ExplorerLinkBuilderProps) => string;
+  reserveFactor: string;
+}
+
+export const ReserveFactorOverview = ({
+  collectorContract,
+  explorerLinkBuilder,
+  reserveFactor,
+}: ReserveFactorOverviewProps) => {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexWrap: 'wrap',
+        justifyContent: 'space-between',
+      }}
+    >
+      <ReserveOverviewBox
+        title={
+          <ReserveFactorTooltip
+            text={<Trans>Reserve factor</Trans>}
+            key="res_factor"
+            variant="description"
+            collectorLink={
+              collectorContract
+                ? explorerLinkBuilder({
+                    address: collectorContract,
+                  })
+                : undefined
+            }
+          />
+        }
+      >
+        <FormattedNumber value={reserveFactor} percent variant="secondary14" visibleDecimals={2} />
+      </ReserveOverviewBox>
+
+      <ReserveOverviewBox
+        title={
+          <Typography variant="description">
+            <Trans>Collector Contract</Trans>
+          </Typography>
+        }
+      >
+        <Link
+          href={explorerLinkBuilder({
+            address: collectorContract,
+          })}
+          target="_blank"
+          sx={{ textDecoration: 'none' }}
+        >
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography variant="description" color="text.secondary">
+              Link
```

Comment:
Could this be better display text for the link?

Reply:
I matched the color from the design.

I actually tested this with `text.primary` instead, and it looked much worse having the link the same color as the text/number fields

Reply:
I was commenting mainly on the verbiage `Link`.  Maybe something like `View Contract` or whatever is most relevant and less generic?